test/instance.ts

```diff
@@ -938,4 +940,218 @@ describe('Instance', () => {
+        parent: instance.formattedName_,
+      });
+
+      instance.request = config => {
+        assert.strictEqual(config.client, 'DatabaseAdminClient');
+        assert.strictEqual(config.method, 'listBackups');
+        assert.deepStrictEqual(config.reqOpts, expectedReqOpts);
+
+        assert.notStrictEqual(config.reqOpts, QUERY);
+        assert.deepStrictEqual(QUERY, ORIGINAL_QUERY);
+
+        assert.strictEqual(config.gaxOpts, QUERY);
+      };
+
+      await instance.getBackups(QUERY);
+    });
+
+    it('should not require a query', async () => {
+      instance.request = config => {
+        assert.deepStrictEqual(config.reqOpts, {
+          parent: instance.formattedName_,
+        });
+
+        assert.deepStrictEqual(config.gaxOpts, {});
+      };
+
+      await instance.getBackups();
+    });
+
+    describe('error', () => {
+      const REQUEST_RESPONSE_ARGS = [new Error('Error.'), null, {}];
+
+      beforeEach(() => {
+        instance.request = (config, callback: Function) => {
+          callback.apply(null, REQUEST_RESPONSE_ARGS);
+        };
+      });
+
+      it('should execute callback with original arguments', done => {
+        instance.getBackups(QUERY, (...args) => {
+          assert.deepStrictEqual(args, REQUEST_RESPONSE_ARGS);
+          done();
+        });
+      });
+    });
+
+    describe('success', () => {
+      const BACKUPS = [
+        {
+          name: 'backup-name',
+          database: 'database-name',
+          expireTime: new PreciseDate(1000),
+        },
+      ];
+
+      // tslint:disable-next-line no-any
+      const REQUEST_RESPONSE_ARGS: any = [null, BACKUPS, {}];
+
+      beforeEach(() => {
+        instance.request = (config, callback) => {
+          callback.apply(null, REQUEST_RESPONSE_ARGS);
+        };
+      });
+
+      it('should create and return Backup objects', done => {
+        const fakeBackupInstance = {};
+
+        instance.backup = backupId => {
+          assert.strictEqual(backupId, BACKUPS[0].name);
+          return fakeBackupInstance as Backup;
+        };
+
+        instance.getBackups(QUERY, (...args) => {
+          assert.ifError(args[0]);
+          assert.strictEqual(args[0], REQUEST_RESPONSE_ARGS[0]);
+          const backup = args[1]!.pop();
+          assert.strictEqual(backup, fakeBackupInstance);
+          assert.strictEqual(args[2], REQUEST_RESPONSE_ARGS[2]);
+          done();
+        });
+      });
+    });
+  });
+
+  describe('backup', () => {
+    const BACKUP_NAME = 'backup-name';
+    const DATABASE_NAME = 'database-name';
+    const EXPIRE_TIME = new PreciseDate({seconds: 3, nanos: 5});
+
+    it('should throw if a backup ID is not provided', () => {
+      assert.throws(() => {
+        instance.backup(null!);
+      }, /A backup ID is required to create a Backup\./);
+    });
+
+    it('should create a Backup instance', () => {
+      const backup = instance.backup(BACKUP_NAME);
+      assert.strictEqual(
+        backup.formattedName_,
```

Comment:
The test shouldn't rely on behavior outside of this method. We should mock the backup file and check that it was called correctly with the 'this' instance and the user-provided name.